packages/dds/tree/src/feature-libraries/chunked-forest/codec/codecs.ts

```diff
@@ -18,15 +18,43 @@ import { TreeCompressionStrategy } from "../../treeCompressionUtils.js";
 
 import { decode } from "./chunkDecoding.js";
 import type { FieldBatch } from "./fieldBatch.js";
-import { EncodedFieldBatch, validVersions } from "./format.js";
+import { EncodedFieldBatch, validVersions, type EncodedFieldBatchFormat } from "./format.js";
 import { schemaCompressedEncode } from "./schemaBasedEncode.js";
 import { uncompressedEncode } from "./uncompressedEncode.js";
 
+export interface IncrementalEncodingParameters {
+	/**
+	 * The data for fields that support incremental encoding during encoding should be stored here.
```

Comment:
I think documenting properties in terms of the semantics of the data in them rather than how they should be used is preferred as its more concise, and also better documents it for both readers and writers.

```suggestion
	 * Fields that support incremental encoding.
```

Reply:
This is removed. I added other interfaces and comments. Please take a look.